docs/crictl.md

```diff
@@ -208,6 +208,24 @@ $ cat pod-config.json
     "linux": {
     }
 }
+```
+
+If the runtime uses systemd as the cgroup driver, set the `"cgroup_parent"` field in the pod-config, similar to
```

Comment:
is this something crictl can check automatically and at least warn about?

Reply:
I think crictl will benefit from this additional check or helping with cgroup configuring. Maybe with special command line flag.

This is not blocking as having this doc update is also useful for sure

Reply:
I will raise a separate PR to discuss on the new flag or functionality to set the correct cgroup_parent values